atlassian_jwt_auth/signer.py

```diff
@@ -9,11 +9,15 @@
 
 class JWTAuthSigner(object):
 
-    def __init__(self, issuer, private_key_retriever, **kwargs):
+    def __init__(self, issuer, private_key_retriever,
```

Comment:
I prefer the current approach of using `**kwargs`. What I feel is lacking is docstring to explain what `kwargs` can be used.